.mvn/maven-build-cache-config.xml

```diff
@@ -71,6 +71,22 @@
                         <goal>install</goal>
                     </goals>
                 </goalsList>
+                <goalsList artifactId="docker-maven-plugin">
```

Comment:
@alerman I'm not sure any of these changes actually help.  I still have issues with the build cache when building everything.